x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/xpack/security/support/SecurityIndexManagerIntegTests.java

```diff
@@ -80,6 +94,55 @@ protected void doRun() throws Exception {
         }
     }
 
+    public void testTemplateSettingsDontAffectSecurityIndex() throws Exception {
```

Comment:
I like this test. Can we also assert that explicit updating the settings is rejected as well? I am aware this is blocked in general for system indices. But the code is away from here. There also have been bugs in how it handles settings that are *not* explicitly configured. So I think an explicit assertion here helps make it clear.

Reply:
I tacked on an explicit settings update (that fails) in the same test, after the index had been created: [6fe4de1](https://github.com/elastic/elasticsearch/pull/97815/commits/6fe4de150ae5de9115bba6369b17dd009e7de928), in order to minimize the overhead of a new test, and changed the test method name [ecb6b77](https://github.com/elastic/elasticsearch/pull/97815/commits/ecb6b77713ea97a3bab7c4668fe506414e073d39) to better reflect the scope of the test.